CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [11.0.1] - 2022-07-14
```

Comment:
I would say this is not a patch change, probably a minor

Reply:
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.

This is not a functionality but a bug fix no? No strong opinion though

Reply:
same here)